diff -ruN common/gnupg/.gnupg/gpg-agent.conf current/gnupg/.gnupg/gpg-agent.conf
--- common/gnupg/.gnupg/gpg-agent.conf	2019-11-17 13:47:27.000000000 +0200
+++ current/gnupg/.gnupg/gpg-agent.conf	1970-01-01 03:00:00.000000000 +0300
@@ -1 +0,0 @@
-enable-ssh-support
diff -ruN common/nvim/.config/nvim/init.vim current/nvim/.config/nvim/init.vim
--- common/nvim/.config/nvim/init.vim	2019-11-17 13:47:27.000000000 +0200
+++ current/nvim/.config/nvim/init.vim	2019-11-16 15:15:43.000000000 +0200
@@ -43,8 +43,6 @@
   \ 'do': 'bash install.sh',
   \ }
 
-Plug 'aserebryakov/vim-todo-lists'
-
 call plug#end()
 
 set laststatus=2
diff -ruN common/tmux/.tmux.conf current/tmux/.tmux.conf
--- common/tmux/.tmux.conf	2019-11-17 13:47:27.000000000 +0200
+++ current/tmux/.tmux.conf	2019-11-16 15:15:43.000000000 +0200
@@ -4,19 +4,23 @@
 set-option -g prefix C-t
 bind-key C-t send-prefix
 
+bind -n M-n send Left
+bind -n M-e send Down
+bind -n M-i send Up
+bind -n M-o send Right
+
 unbind '"'
 unbind %
 bind | split-window -h -c "#{pane_current_path}"
 bind - split-window -v -c "#{pane_current_path}"
 bind c new-window -c "#{pane_current_path}"
 
-#set -g base-index 1
+set -g base-index 1
 set-option -g allow-rename off
 setw -g mode-keys vi
 
 set-window-option -g utf8 on
 set -g default-terminal "screen-256color"
 set-option -g default-shell /bin/zsh
-set -g default-command "${SHELL}"
 
 set -g escape-time 0
diff -ruN common/zsh/.zlogin current/zsh/.zlogin
--- common/zsh/.zlogin	2019-11-17 13:47:27.000000000 +0200
+++ current/zsh/.zlogin	1970-01-01 03:00:00.000000000 +0300
@@ -1 +0,0 @@
-eval $(gpg-agent --daemon)
diff -ruN common/zsh/.zshenv current/zsh/.zshenv
--- common/zsh/.zshenv	2019-11-17 13:47:27.000000000 +0200
+++ current/zsh/.zshenv	1970-01-01 03:00:00.000000000 +0300
@@ -1,4 +0,0 @@
-source ~/vinted/dev-tools/docker/aliases
-
-GPG_TTY=$(tty)
-export GPG_TTY
diff -ruN common/zsh/.zshrc current/zsh/.zshrc
--- common/zsh/.zshrc	2019-11-17 13:47:27.000000000 +0200
+++ current/zsh/.zshrc	2019-11-16 15:15:43.000000000 +0200
@@ -35,11 +35,4 @@
 
 alias dodo='docker run --rm -it -v "$PWD:$PWD" -w "$PWD" -u $(id -u):$(id -g)'
 
-[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
-bindkey -r "^T"
-bindkey -r "^R"
-bindkey "^F" fzf-file-widget
-bindkey "^H" fzf-history-widget
-
-SSH_AUTH_SOCK=$(gpgconf --list-dirs agent-ssh-socket)
-export SSH_AUTH_SOCK
+#[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
